docs/use-cases/airdrop.mdx

```diff
@@ -0,0 +1,48 @@
+---
+<!-- TODO: description --->
```

Comment:
Add it now (up to 160 characters). Also, add a `slug: /use-cases/passport-enabled-airdrops`